src/bokeh/models/widgets/tables.py

```diff
@@ -858,6 +858,47 @@ def __init__(self, *args, **kwargs) -> None:
     The height of each row in pixels.
     """)
 
+    @staticmethod
+    def from_data(data, columns=None, formatters={}, **kwargs) -> DataTable:
+        """ Create a simple table from a pandas dataframe, dictionary or ColumnDataSource.
+
+        Args:
+            data (DataFrame or dict or ColumnDataSource) :
+                The data to create the table from. If the data is a dataframe or dictionary, it will be converted to a ColumnDataSource
+
+            columns (list, optional) :
+                A list of column names to use. If None, use all columns
+
+            formatters (dict, optional) :
+                Formatters to be applied to specified columns
```

Comment:
I'd suggest being a little more detailed about what is expected from this argument
```suggestion
            formatters (dict, optional) :
                A mapping of column names and corresponding Formatters to
                apply to each column.
```